src/main/com/marginallyclever/makelangelo/paper/PaperMenuItem.java

```diff
@@ -0,0 +1,27 @@
+package com.marginallyclever.makelangelo.paper;
+
+import javax.swing.*;
+
+public class PaperMenuItem extends JRadioButtonMenuItem {
+
+    private final PaperSize paperSize;
+    private final PaperSettings paperSettings;
+
+    public PaperMenuItem(PaperSettings paperSettings, PaperSize paperSize) {
+        super(paperSize.toString());
+        this.paperSettings = paperSettings;
+        this.paperSize = paperSize;
+        addActionListener((e)-> {
+            paperSettings.changePaperSize(paperSize);
+        });
+    }
+
+    public boolean updateSelected() {
+        double w = paperSettings.getCurrentPaper().getPaperWidth();
+        double h = paperSettings.getCurrentPaper().getPaperHeight();
+        boolean match = paperSize.width == w && paperSize.height == h;
```

Comment:
il me semble manquer le cas quand l'orientation du papier est modifier via la checkBox Landscap  "⇋"

Reply:
ok, I'll wait for #367 to be fixed in order to add this feature